packages/web-app-admin-settings/src/composables/useAdminUsersFlags.ts

```diff
@@ -0,0 +1,20 @@
+import { computed, type ComputedRef } from 'vue'
+import { storeToRefs } from 'pinia'
+import { useCapabilityStore } from '@opencloud-eu/web-pkg'
+
+type Caps = {
+  graph?: { users?: { edit_login_allowed_disabled?: boolean } }
+}
+
+export function useAdminUsersFlags(): {
+  isLoginToggleHidden: ComputedRef<boolean>
+} {
+  const capabilityStore = useCapabilityStore()
+  const { capabilities } = storeToRefs(capabilityStore)
+
+  const isLoginToggleHidden = computed<boolean>(() => {
+    return (capabilities.value as Caps)?.graph?.users?.edit_login_allowed_disabled === true
```

Comment:
Please add `edit_login_allowed_disabled` to the `Capabilities` interface in `packages/web-client/src/ocs/capabilities.ts`. Then you can remove `Caps` here.

Also I think you should add a default value in `packages/web-pkg/src/composables/piniaStores/capabilities.ts` and a computed in the store in the same file (e.g. `graphUsersEditLoginAllowedDisabled`) - check the others for examples.